Source/WebCore/rendering/svg/RenderSVGText.cpp

```diff
@@ -415,8 +408,15 @@ void RenderSVGText::layout()
     if (isLayerBasedSVGEngineEnabled()) {
         updateLayerTransform();
         updateCachedBoundariesInParents = false; // No longer needed for LBSE.
-    } else if (!updateCachedBoundariesInParents)
-        updateCachedBoundariesInParents = oldBoundaries != objectBoundingBox();
+    } else {
+        if (m_needsTransformUpdate) {
+            m_localTransform = textElement().animatedLocalTransform();
```

Comment:
```
AffineTransform SVGTextElement::animatedLocalTransform() const
{
    AffineTransform matrix;
    auto* style = renderer() ? &renderer()->style() : nullptr;

    // if CSS property was set, use that, otherwise fallback to attribute (if set)
    if (style && style->hasTransform()) {
        TransformationMatrix t;
        // For now, the transform-origin is not taken into account
        // Also, any percentage values will not be taken into account
        style->applyTransform(t, FloatRect(0, 0, 0, 0), RenderStyle::individualTransformOperations);
        // Flatten any 3D transform
        matrix = t.toAffineTransform();
    } else
        matrix = transform().concatenate();

    const AffineTransform* transform = const_cast<SVGTextElement*>(this)->supplementalTransform();
    if (transform)
        return *transform * matrix;
    return matrix;
}
```

It looks like this function uses style, but style should already have been computed before `layout()` is called, so that part of this change should have no effect.

It also looks like `supplementalTransform()` is the only code in here that could read layout information, but it looks like `supplementalTransform()` is only set for SMIL motion animation. However, `imported/w3c/web-platform-tests/css/css-transforms/transform-box/view-box-mutation-002.html` doesn't seem to be using such an element. So I'm a bit confused about how this works